scripts/trainer.py

```diff
@@ -34,13 +26,7 @@ def compress_file(self, path):
 
     def download_db(self, db_type):
         path = f"data/{db_type}.json"
```

Comment:
Instead of passing the path manually, we should use bugzilla.BUGS_DB instead of "bugs" and repository.COMMITS_DB instead of "commits".

Reply:
I was trying to avoid unnecessary imports. But yes, it gives more structure to the code. I'll do it.

Reply:
They are not unnecessary, as we want to use them to reduce code duplication